internal/graph/reachableresources.go

```diff
@@ -47,7 +47,10 @@ func (crr *ConcurrentReachableResources) ReachableResources(
 	sort.Strings(req.SubjectIds)
 
 	limits, ctx := newLimitTracker(stream.Context(), req.OptionalLimit)
-	ci := newCursorInformation(req.OptionalCursor, limits)
+	ci, err := newCursorInformation(req.OptionalCursor, req.Revision, limits)
```

Comment:
So essentially the client is responsible to make sure they provide the corresponding revision used for the cursor?
I wonder if this might end up being prone to error and confuse our users, which might not be used to using ZedTokens at all if they use `minimize_latency` semantics.

Did you consider setting the request revision to what's in the cursor if the user uses `minimize_latency` and provide no ZedToken? Can we add tests for all this?

Reply:
No, the client is not responsible. The revision is encoded into the cursor and the consistency middleware extracts it. This check here is an *internal* check to make sure the consistency middleware pulled out the correct revision and to ensure we have it for minting new cursors

> Did you consider setting the request revision to what's in the cursor if the user uses minimize_latency and provide no ZedToken? Can we add tests for all this?

This is what happens now and yes, we have tests for it

Reply:
yeah sorry, that change came in a commit later so it wasn't reviewed "transactionally" 😅